### ISSUE TITLE
Crash after update: java.lang.NullPointerException: accountManager.getUserDa…kInfo.KEY_DECSYNC_DIR_ID) must not be null

### ISSUE THREAD

**Issue by gAQeNQ** [39aldo39/DecSyncCC#79]
After update from `2.1.0` to `2.2.0`:

1.  I was asked to select folder again.
2. I have selected previous folder
3. After hitting sync button app crashes with this error

```
12-29 19:44:05.279 27669 30298 D DecSyncCC: Load unknown address books
12-29 19:44:05.330 27669 30298 E AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-2
12-29 19:44:05.330 27669 30298 E AndroidRuntime: Process: org.decsync.cc, PID: 27669
12-29 19:44:05.330 27669 30298 E AndroidRuntime: java.lang.NullPointerException: accountManager.getUserDa…kInfo.KEY_DECSYNC_DIR_ID) must not be null
12-29 19:44:05.330 27669 30298 E AndroidRuntime: 	at org.decsync.cc.ui.MainActivity.getUnknownBooks(MainActivity.kt:6)
12-29 19:44:05.330 27669 30298 E AndroidRuntime: 	at org.decsync.cc.ui.MainActivity.loadBooksUnknown(MainActivity.kt:2)
12-29 19:44:05.330 27669 30298 E AndroidRuntime: 	at org.decsync.cc.ui.MainActivity.loadCollections(MainActivity.kt:3)
12-29 19:44:05.330 27669 30298 E AndroidRuntime: 	at org.decsync.cc.ui.MainActivity$onCreate$4.invokeSuspend(MainActivity.kt:12)
12-29 19:44:05.330 27669 30298 E AndroidRuntime: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:4)
12-29 19:44:05.330 27669 30298 E AndroidRuntime: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:18)
12-29 19:44:05.330 27669 30298 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:1)
12-29 19:44:05.330 27669 30298 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:4)
12-29 19:44:05.330 27669 30298 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:4)
12-29 19:44:05.330 27669 30298 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:1)
```

Downgrade to `2.1.0` solved the issue.

**Comment by nubiai** [comment]
Same here.

**Comment by mitchhitch** [comment]
Same here OP5t running lineage 18.1

FATAL EXCEPTION: DefaultDispatcher-worker-2
Process: org.decsync.cc, PID: 9640
java.lang.NullPointerException: accountManager.getUserDa…kInfo.KEY_DECSYNC_DIR_ID) must not be null
	at org.decsync.cc.ui.MainActivity.getUnknownBooks(MainActivity.kt:6)
	at org.decsync.cc.ui.MainActivity.loadBooksUnknown(MainActivity.kt:2)
	at org.decsync.cc.ui.MainActivity.loadCollections(MainActivity.kt:3)
	at org.decsync.cc.ui.MainActivity$onCreate$4.invokeSuspend(MainActivity.kt:12)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:4)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:18)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:1)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:4)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:4)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:1)

**Comment by 39aldo39** [comment]
Yes, I noticed it as well, it happens only on Android 11. It is fixed in v2.2.2. (Note that it is not fixed in v2.2.1.)
